recipe/meta.yaml

```diff
@@ -19,14 +19,19 @@ build:
 requirements:
   build:
     - python         # [win]
```

Comment:
Remove this line

Reply:
I didn't add this, so I'm not sure if I should be the one to remove it in this PR. @jakirkham what do you think?